### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by flippinjoe21**
Cleans up the proxy stuff a little bit, and adds some more example of using the autosizing

### CODE REVIEW

File: Pod/Classes/URBNDataSourceAdapter.m [hunk 1]
```diff
@@ -45,23 +46,30 @@ @interface URBNDataSourceInternalResponder : NSObject
...
+
 
 @interface URBNDataSourceAdapter ()
 
-@property (nonatomic, strong) URBNDSICollectionDelegate *internalCollectionDelegate;
-@property (nonatomic, strong) URBNDSITableDelegate *internalTableDelegate;
+// AutoSizing
+@property (nonatomic, strong) URBNDSISizingDelegate *autoSizingDelegate;
+@property (nonatomic, strong) NSMutableDictionary *prototypeCells;
+@property (nonatomic, strong) NSMutableDictionary *prototypeHeaders;
```
Comment (jgrandelli) [inline Pod/Classes/URBNDataSourceAdapter.m]: I assume this is actually headers and footers?

Comment (flippinjoe21) [inline Pod/Classes/URBNDataSourceAdapter.m]: named correctly now

**Comment by jgrandelli** [comment]
Supple. Nice

**Comment by jgrandelli** [comment]
this is much better

**Pull request merged**